osf/management/commands/spam_metrics.py

```diff
@@ -0,0 +1,113 @@
+import csv
+import pytz
+import tempfile
+import datetime
+import logging
+import requests
+
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+from framework.celery_tasks import app as celery_app
+from osf.models import NodeLog, OSFUser, SpamStatus, PreprintLog
+
+from website import settings
+
+logger = logging.getLogger(__name__)
+
+
+@celery_app.task(name='osf.management.commands.spam_metrics')
+def gather_metrics(dry_run=False):
+    today = timezone.now().today()
+    first = today.replace(day=1)
+    last_month = first - datetime.timedelta(days=1)
+    last_month.replace(tzinfo=pytz.utc)
+
+    data = {
+        # Node Log entries
+        'confirmed_spam_node': NodeLog.objects.filter(
+            action=NodeLog.CONFIRM_SPAM,
+            created__gt=last_month,
```

Comment:
This ought be runnable later in case of failure, in which case each of these needs both `created__gte=prior_month` and `created__lt=target_month`